spotify_player/src/cli/client.rs

```diff
@@ -130,6 +141,16 @@ async fn handle_socket_request(
 
             Ok(Vec::new())
         }
+        Request::Playlist(command) => {
+            let resp = handle_playlist_request(client, state, command).await?;
+
+            // Allowing command output of success
+            let mut resp_vec = serde_json::to_vec_pretty(&resp)?;
+            // Removing quotation marks
+            resp_vec.pop();
+            resp_vec.remove(0);
```

Comment:
The `resp` is a string, why don't use `String::into_bytes` instead of converting from  `json` and manually removing the quotation marks?